MAINTENANCE.md

```diff
@@ -0,0 +1,75 @@
+
+## Procedures
+To ensure proper health of the infrastructure, the Tsar regularly monitors its proper operation. If a malfunction is detected, it is the Tsar's job to ensure that there’s someone working on it (or solve it on their own).
+
+If it is unclear what area an issue belongs to, the Tsar asks for advice from other team members, on Slack or GitHub. If, after asking for advice, it turns out that nobody in the team knows how to classify it, the issue must be classified with a “stat:needs triage” label.
+
+If it is unclear who should review an external PR, the Tsar asks for advice from the rest of the core team. If after asking for advice, it is still unclear who should do it, the reviewer for such a PR will be decided at the next Dotty meeting.
+
+In general, if anything else is unclear for proper fulfilment of responsibilities, the Tsar must proactively seek advice from other team members on Slack or other channels.
+
+## Reporting
+At the end of their Tsardom, the Tsar reports to the team during the Dotty meeting on the following points:
+
+- Whether there were any incidents with the CI, nightlies and benchmarks, how they were resolved and what steps were taken to prevent them from happening in the future.
+- How many new external contributors’ PRs were there and what they were about (in brief).
+- How many new issues were opened during their Tsardom period? Were there any areas that got a lot of issues? How many regressions from a prior Scala 3 release were there? Which were designated for an MSc project or an Issue Spree?
+- If new labels were created or old ones were removed, or there is any other feedback on how to improve the Tsardom, mention that.
+- Unassigned PRs and issues that the team failed to classify: bring them one by one so that the team can make a decision on them.
+
+# Maintenance List
+The following is the list of all the principal areas of the compiler and the core teams members who are responsible for their maintenance:
+
+- Typer: @odersky, @smarter, (@dwijnand)
+- Erasure: @smarter, @odersky
+- Enums: @bishabosha
+- Pattern Matching: @dwijnand, (@liufengyun), @sjrd
+- Inline: @nicolasstucki, @odersky
+- Metaprogramming (Quotes, Reflect, Staging): @nicolasstucki, @aherlihy
+- Match types: @OlivierBlanvillain, @dwijnand
+- GADT: @abgruszecki, @dwijnand
+- Scaladoc: @KacperFKorban,  @BarkingBad, @pikinier20
+- Initialization checker: @olhotak, @liufengyun, @anatoliykmetyuk
+- Safe nulls: @noti0na1, @olhotak
+- tailrec: @sjrd, @mbovel
+- JS backend: @sjrd
+- forward compat (-scala-release): @prolativ, @Kordyjan, (@nicolasstucki)
+- Benchmarks: @anatoliykmetyuk, @mbovel
+- REPL: Dale, @anatoliykmetyuk, @prolativ
+- CI: @anatoliykmetyuk
+- Community Build: @anatoliykmetyuk
+- Vulpix: Dale, @prolativ
+- JVM backend: @sjrd (review only), @Kordyjan
+- Derivation & Mirrors: @bishabosha, (@dwijnand)
+- Linting (especially unused warnings) / Reporting UX  : VirtusLab TBD?
+- Java-compat: @Kordyjan
```

Comment:
/cc @bishabosha @liufengyun @aherlihy @OlivierBlanvillain @abgruszecki @KacperFKorban @BarkingBad @pikinier20 @olhotak @noti0na1 @Kordyjan 

CC'ing everyone who wasn't on the meeting where the list was composed – please feel free to submit your corrections if we got something wrong.